tests/test_transforms/test_math/test_binary_operator_transform.py

```diff
@@ -0,0 +1,339 @@
+import operator
```

Comment:
There isn't any test on checking that we create column with a name if `out_column` isn't set.

Reply:
I added these tests to test_inference

Reply:
I think it should also be here because we can generate columns even if there is no `out_column` set.